src/AutoRest.CSharp/Common/Input/CodeModelInputs.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using AutoRest.CSharp.Input;
+using Azure.Core;
+
+#pragma warning disable SA1649
+namespace AutoRest.CSharp.Common.Input
+{
+    internal record CodeModelOperation(
```

Comment:
Its very awkward to only update the Operation and not the OperationGroup.  It creates a situation where mgmt plane code needs to deal with both CodeModelClasses and the new InputClasses which is not ideal at all.  Lets look at refactoring the cut over here so we don't need to deal with both types depending on which class we are in.

Reply:
`CodeModelOperation` is removed.

However, MPG and DPG share some good chunks of code that has been affected by the change. The biggest one is probably `RestClientBuilder` that is going to work with InputClasses. It is used by `MgmtRestClient` and `MgmtOutputLibrary`. The only way to isolate MPG from InputClasses here is to create duplication of good chunk of functionality.